network_setup.sh

```diff
@@ -100,6 +100,30 @@ function check_connection {
 	fi
 }
 
+function check_ssh_host_keys() {
+	# Sometimes these files are empty after first boot and ssh won't work.
```

Comment:
Could you add the links to github issues about others that ran intro similar issues.

Reply:
Added!